src/release.ts

```diff
@@ -10,28 +60,46 @@ const defaultPrereleaseTag = 'rc';
 
 prompt.selectOption;
 
+export enum ReleaseType {
+  Release,
+  ReleaseCandidate,
+  BasePrerelease,
+}
+
+export async function tagBasePrerelease(prereleaseTag = defaultPrereleaseTag, releaseBranch = defaultReleaseBranch) {
```

Comment:
just wondering if we should have the default prerelease tag  (RC) here. It might be better to just fail if not provided. Then in the buildsystem we can prompt for a PR number or other text